### PR TITLE
Bug fix : Red Task issue

### PR DESCRIPTION
**Pull request by ardourApeX**
Date: `15-Dec-2023`

Developer Name: @ardourApeX 

----

## Issue Ticket Number

-  https://github.com/Real-Dev-Squad/website-my/issues/543

## Description

Fixed red task issue for both dev flag enabled & disabled.

**Documentation Updated?**

-   [ ] Yes
-   [x] No

**Breaking Changes**

-   [ ] Yes
-   [x] No

**Development Tested?**

-   [x] Yes
-   [ ] No

**Under Feature Flag**

-   [ ] Yes
-   [x] No

**Database Changes**

-   [ ] Yes
-   [x] No

### Screenshots

https://github.com/Real-Dev-Squad/website-my/assets/56365512/7b96b87e-aeaa-4def-973d-9959f6a955e6

## Test Coverage
Screenshot 2023-12-15 at 8 29 41 AM

## Additional Notes
NA

### CODE REVIEW

**Review by ajoykumardas12 — APPROVED** [review]
LGTM

File: app/components/task/holder.js [hunk 1]
```diff
@@ -14,6 +14,20 @@ export default class TasksHolderComponent extends Component {
...
+  constructor() {
+    super(...arguments);
+    if (this.args.dev) {
+      this.status =
+        this.args.task.status === TASK_KEYS.COMPLETED
+          ? TASK_KEYS.DONE
+          : this.args.task.status;
+    } else {
+      this.status =
+        this.args.task.status === TASK_KEYS.DONE
```
Comment (vikhyat187) [inline app/components/task/holder.js:26]: So just to confirm we are migrating all the `COMPLETED` statuses to `DONE`?

Comment (prakashchoudhary07) [reply app/components/task/holder.js:26]: In some states where something is done and others are completed, only one would show.

Comment (ardourApeX) [reply app/components/task/holder.js:26]: > So just to confirm we are migrating all the `COMPLETED` statuses to `DONE`?

Yes

Comment (ardourApeX) [reply app/components/task/holder.js:26]: > In some states where something is done and others are completed, only one would show.

Yes. Reason being with dev mode enabled we have only `DONE` status and without dev mode being enabled we only consider `COMPLETED`.

Comment (vikhyat187) [reply app/components/task/holder.js:26]: Understood

**Review by vikhyat187 — APPROVED** [review]

**Pull request merged**